packages/use-media/src/utilities/queryObjectToString.ts

```diff
@@ -0,0 +1,30 @@
+import { MediaQueryObject } from '../types'
+import camelToHyphen from './camelToHyphen'
+
+const QUERY_COMBINATOR = ' and '
+
+export default function queryObjectToString(query: string | MediaQueryObject) {
```

Comment:
Could we force media query string only ? Less maintenance and better aligned with standards maybe ?

Reply:
We mostly use `useMedia` with an object property, so we'll need to update it. `MediaQueryObject` doesn't provide useful completions, so I agree that it may be a good idea to only use strings